code/modules/surgery/organs/mmi_holder.dm

```diff
@@ -23,8 +25,6 @@
 			stored_mmi.forceMove(get_turf(owner))
 			stored_mmi = null
 	..()
```

Comment:
Some of these return the value from the parent call, some don't. I think you should make it consistent by returning it always instead of sometimes.

```suggestion
	return ..()
```